.github/workflows/benchmark-runner.yml

```diff
@@ -0,0 +1,131 @@
+on:
+  workflow_call:
+    inputs:
+      setup:
+        required: true
+        type: string
+      architecture:
+        required: true
+        type: string
+      instance-type:
+        required: true
+        type: string
+      ami-id:
+        required: true
+        type: string
+      github-runner-label:
+        required: true
+        type: string
+    secrets:
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+      AWS_REGION_BENCHMARK:
+        required: true
+      AWS_EC2_SG_ID_BENCHMARK:
+        required: true
+      AWS_EC2_SUBNET_ID_BENCHMARK:
+        required: true
+      GH_PERSONAL_ACCESS_TOKEN:
+        required: true
+      PERFORMANCE_RTS_HOST:
+        required: true
+      PERFORMANCE_RTS_PORT:
+        required: true
+      PERFORMANCE_RTS_AUTH:
+        required: true
```

Comment:
Can we inherit the secrets (second level generation) somehow? @GuyAv46 Seems like this is not working out of the box as we see for regular "secrets generation"